ckan/migration/versions/085_adjust_activity_timestamps.py

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+
+import pytz
+import tzlocal
+import logging
+
+log = logging.getLogger(__name__)
+
+def upgrade(migrate_engine):
+"""
+The script assumes that the current timestamp was recorded with the server's current set timezone
+"""
+    local_tz = tzlocal.get_localzone()
+
+    with migrate_engine.begin() as connection:
+        sql = "select id, timestamp from activity"
+        results = connection.execute(sql)
+
+        log.info("Adjusting Activity timestamp, server's localtime zone: {tz}".format(tz=local_tz))
+
+        for row in results:
```

Comment:
I tried doing this all in sql, but one issue i ran into was getting the system's set timezone. Another problem is, it's been awhile since I've had to write some complex sql =p.